tutorials/getting_started/another_model_example.md

```diff
@@ -0,0 +1,606 @@
+data in the same way that the model's training data was processed, and this redirection makes that
+possible.  Here's `text_to_instance`:
+
+```python
+    def text_to_instance(self, title: str, abstract: str, venue: str = None) -> Instance:
+        tokenized_title = self._tokenizer.tokenize(title)
+        tokenized_abstract = self._tokenizer.tokenize(abstract)
+        title_field = TextField(tokenized_title, self._token_indexers)
+        abstract_field = TextField(tokenized_abstract, self._token_indexers)
+        fields = {'title': title_field, 'abstract': abstract_field}
+        if venue is not None:
+            fields['label'] = LabelField(venue)
+        return Instance(fields)
+```
+
+The first important thing to notice here is that an `Instance` is made up of a named collection of
+`Fields`.  These `Fields` get converted by the rest of the AllenNLP library into batched arrays
+that get passed to your `Model`.  A `TextField` represents a tokenized piece of text, a
+`LabelField` represents a categorical label, and there are other `Fields` representing other kinds
+of data that we don't need to use here.  These inputs are plain strings, but they will be converted
+to integer arrays before being passed to your `Model` after building a vocabulary from the training
+data.  All of this is handled for you by AllenNLP, in a way that you can configure.
+
+The second important thing to notice is that we're using some objects on `self` to configure how
+the `TextFields` are constructed: `self._tokenizer` and `self._token_indexers`.  The `Tokenizer`
+determines how the title and abstract get converted into tokens.  The `Tokenizer` could split the
+string into words, characters, byte pairs, or any other way you want to tokenize your data.  The
+`TokenIndexers` then determine how to represent these tokens as arrays in your model.  If your
+tokens are words, for instance, the `TokenIndexers` could make arrays out of the word id, the
+characters in the word, the word's part of speech, or many other things.  Notice that we're not
+specifying _any_ of that in the code, we're just saying that we'll get a `Tokenizer` and some
+`TokenIndexers` as input to this `DatasetReader`, and we'll use them to process our data.  This
+lets us be _really_ flexible later in how exactly our inputs get represented.  We can experiment
+with using character-level CNNs or POS-tag embeddings without changing our `DatasetReader` or our
+`Model` code at all.
+
+How does this work?  This brings us to the last pieces of our `DatasetReader`: the constructor and
+the `from_params` method.
+
+```python
+@DatasetReader.register("s2_papers")
+class SemanticScholarDatasetReader(DatasetReader):
+    def __init__(self,
+                 tokenizer: Tokenizer = None,
+                 token_indexers: Dict[str, TokenIndexer] = None) -> None:
+        self._tokenizer = tokenizer or WordTokenizer()
+        self._token_indexers = token_indexers or {"tokens": SingleIdTokenIndexer()}
```

Comment:
You could mention what `SingleIdTokenIndexer` does.

Reply:
The text currently says,

```
The constructor takes these dependencies as inputs, where by default we're splitting text into  
words and representing them as single word ids (as opposed to, say, sequences of character ids).
```

Is that not enough, given the description in the paragraph above?  How would you word this?